src/check-pr-action.ts

```diff
@@ -168,14 +168,19 @@ async function main() {
             if (requireLabel) {
                 core.setFailed(lines.join("\n"));
             }
-        } else if (change.changeType === ChangeType.TASK) {
-            lines.push(
-                "This change is marked as an *internal change* (Task), so will not be included in the changelog.",
-            );
-        } else if (change.notes == null) {
-            lines.push(
-                "This change has no change notes, so will not be included in the changelog.",
-            );
+        } else if (change.notes == null || change.changeType === ChangeType.TASK) {
+            if (change.breaking) {
+                lines.push("This change is marked as *breaking*, but is missing a changelog entry.");
```

Comment:
Sorry about the back and forth, but if it is Task + Breaking then it will be correctly included in the changelog (in the breaking group), the issue was only around Task + !Breaking. I think this whole change might be as simple as adding a case for `change.notes == null && change.breaking` above the check for `T-Task` and there failing the job with a helpful message, leaving the Task & no notes cases from before